File: QXQP-C# MVC/QXQPS/Vo/CollageVo.cs
using QXQPS.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace QXQPS.Vo
{
    public class CollageVo: PW_Reception
    {
        public int CollageID { get; set; }//领料ID
        public string ForeMan { get; set; }//领料员
        public string Auditor { get; set; }//审核人
        public string Operator { get; set; }//操作人
        public string Remark { get; set; }//备注
        public string StrAudit { get; set; }//备注


        private string collageDate;//领料日期
        public string CollageDate
        {
            set
            {
                try
                {
                    DateTime dt = Convert.ToDateTime(value);
                    collageDate = dt.ToString("yyyy-MM-dd");
                }
                catch (Exception)
                {

                    collageDate = value;
                }
            }
            get
            {
                return collageDate;
            }
        }//领料日期

        private string auditDate;//审核日期
        public string AuditDate
        {
            set
            {
                try
                {
                    DateTime dt = Convert.ToDateTime(value);
                    auditDate = dt.ToString("yyyy-MM-dd HH:mm:ss");
                }
                catch (Exception)
                {

                    auditDate = value;
                }
            }
            get
            {
                return auditDate;
            }
        }//审核日期

    }
}

File: QXQP-SSM/src/main/java/com/koi/vo/CollageVo.java
package com.koi.vo;

import java.sql.Timestamp;
import java.util.Date;

import com.koi.util.Util;

public class CollageVo {
	private Integer collageId;
	private String foreMan;
	private Integer receptionId;
	private String collageDate;
	private Double amount;
	private Boolean toAudit;
	private Timestamp auditDate;
	private String auditor;
	private String remark;
	private String operator;

	private String carNum;
	private String owner;
	private String maintenanceNum;
	private String collageState;
	private String StrAudit;
	public CollageVo() {
	}

	public CollageVo(Integer collageId, String foreMan, Integer receptionId,
			String collageDate, Double amount, Boolean toAudit, Timestamp auditDate,
			String auditor, String remark, String operator, String carNum,
			String owner, String maintenanceNum, String collageState) {
		super();
		this.collageId = collageId;
		this.foreMan = foreMan;
		this.receptionId = receptionId;
		this.collageDate = collageDate;
		this.amount = amount;
		this.toAudit = toAudit;
		this.auditDate = auditDate;
		this.auditor = auditor;
		this.remark = remark;
		this.operator = operator;
		this.carNum = carNum;
		this.owner = owner;
		this.maintenanceNum = maintenanceNum;
		this.collageState = collageState;
	}

	public Integer getCollageId() {
		return collageId;
	}

	public void setCollageId(Integer collageId) {
		this.collageId = collageId;
	}

	public String getForeMan() {
		return foreMan;
	}

	public void setForeMan(String foreMan) {
		this.foreMan = foreMan;
	}

	public Integer getReceptionId() {
		return receptionId;
	}

	public void setReceptionId(Integer receptionId) {
		this.receptionId = receptionId;
	}

	public String getCollageDate() {
		return collageDate;
	}

	public void setCollageDate(String collageDate) {
		this.collageDate = collageDate;
	}

	public Double getAmount() {
		return amount;
	}

	public void setAmount(Double amount) {
		this.amount = amount;
	}

	public Boolean getToAudit() {
		return toAudit;
	}

	public void setToAudit(Boolean toAudit) {
		this.toAudit = toAudit;
	}

	public Date getAuditDate() {
		return Util.strToUtilDate(this.auditDate,"yyyy-MM-dd HH:mm:ss");
	}

	public void setAuditDate(Timestamp auditDate) {
		this.auditDate = auditDate;
	}

	public String getAuditor() {
		return auditor;
	}

	public void setAuditor(String auditor) {
		this.auditor = auditor;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public String getOperator() {
		return operator;
	}

	public void setOperator(String operator) {
		this.operator = operator;
	}

	public String getCarNum() {
		return carNum;
	}

	public void setCarNum(String carNum) {
		this.carNum = carNum;
	}

	public String getOwner() {
		return owner;
	}

	public void setOwner(String owner) {
		this.owner = owner;
	}

	public String getMaintenanceNum() {
		return maintenanceNum;
	}

	public void setMaintenanceNum(String maintenanceNum) {
		this.maintenanceNum = maintenanceNum;
	}

	public String getCollageState() {
		return collageState;
	}

	public void setCollageState(String collageState) {
		this.collageState = collageState;
	}

	public String getStrAudit() {
		return StrAudit;
	}

	public void setStrAudit(String strAudit) {
		StrAudit = strAudit;
	}
	
}
